cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/OperationsDashboardHttpHandlerTest.java

```diff
@@ -222,46 +228,85 @@ public void testDashboardWithNamespaceFiltering() throws Exception {
 
   }
 
+  List<Long> getExpectedRuntimes(long startTimeInMin, long endTimeInMin, long triggerTimeInMin,
+                                 long actualScheduleStartTime) {
+    List<Long> triggerringTimeInSeconds = new ArrayList<>();
+    for (long currentTimeInMin = startTimeInMin; currentTimeInMin < endTimeInMin; currentTimeInMin += 1) {
+      if (currentTimeInMin % triggerTimeInMin == 0) {
+        // triggering minute
+        long timestampInSeconds = TimeUnit.MINUTES.toSeconds(currentTimeInMin);
+        if (timestampInSeconds >= actualScheduleStartTime) {
+          // for the first triggering minute its possible it is before the actual schedule start time,
```

Comment:
Same here - replace `actual schedule start time` with `test start time`, and below too